src/submission/submission-service.ts

```diff
@@ -504,6 +508,7 @@ export namespace operations {
     const validateResult: ClinicalTypeValidateResult = await validateSubmissionData(
       clinicalSubmissionRecords,
       relevantDonorsMap,
+      command.programId,
```

Comment:
why do we need to pass `command.programId` now, does it somehow use it??

Reply:
Nope I think I added it here first before moving the programId validation logic to upload